src/app/slotting/edit/edit.service.ts

```diff
@@ -48,6 +48,43 @@ export class EditService {
     return ['zeus', 'air', 'armor', 'art', 'hq', 'inf', 'maint', 'mech_inf', 'med', 'mortar', 'motor_inf', 'plane', 'recon', 'service', 'support', 'uav'];
   }
 
+  public get slotPresets(): Array<{ shortcode: string, description: string }> {
+    return [
+      { shortcode: 'CL', description: 'Company Leader' },
+      { shortcode: 'PTL', description: 'Platoon Leader' },
+      { shortcode: 'PTA', description: 'Asst. Platoon Leader' },
+      { shortcode: 'PTM', description: 'Platoon Medic' },
+      { shortcode: 'SQL', description: 'Squad Leader' },
+      { shortcode: 'SQA', description: 'Asst. Squad Leader' },
+      { shortcode: 'SQM', description: 'Squad Medic' },
+      { shortcode: 'FTL', description: 'Fireteam Leader' },
+      { shortcode: 'LAT', description: 'Light Anti-Tank Gunner' },
+      { shortcode: 'AT', description: 'Anti-Tank Gunner' },
+      { shortcode: 'AAT', description: 'Asst. Anti-Tank Gunner' },
+      { shortcode: 'AAA', description: 'Asst. Anti-Air Gunner' },
+      { shortcode: 'AA', description: 'Anti-Air Gunner' },
+      { shortcode: 'MG', description: 'Machine Gunner' },
+      { shortcode: 'AMG', description: 'Asst. Machine Gunner' },
+      { shortcode: 'AAR', description: 'Asst. Autorifleman' },
+      { shortcode: 'AR', description: 'Autorifleman' },
+      { shortcode: 'M', description: 'Medic' },
+      { shortcode: 'GRN', description: 'Grenadier' },
+      { shortcode: 'JTAC', description: 'Joint Terminal Attack Controller' },
+      { shortcode: 'Z', description: 'Zeus' },
+      { shortcode: 'DRV', description: 'Driver' },
+      { shortcode: 'GUN', description: 'Gunner' },
+      { shortcode: 'CMD', description: 'Commander' },
+      { shortcode: 'CMA', description: 'Asst. Commander' },
+      { shortcode: 'RTO', description: 'Radiotelephone Operator' },
+      { shortcode: 'E', description: 'Engineer' },
+      { shortcode: 'DM', description: 'Designated Marksman' },
+      { shortcode: 'CFR', description: 'Combat First Responder' },
+      { shortcode: 'P', description: 'Pilot' },
+      { shortcode: 'CP', description: 'Co-Pilot' },
+      { shortcode: 'R', description: 'Rifleman' }
+    ].sort((a, b) => a.description.localeCompare(b.description));
```

Comment:
gibt's eine innere Logik in der Unsortiertheit der Liste? Sonst sortier die Zeilen doch auch im Code alphabetisch, dann brauchst du das `sort` nicht.

Reply:
Ja und dann fügt wieder jemand irgendwas hinzu und achtet ned drauf. 😅

Reply:
dazu gibts CR